internal/pkg/util/fs/squashfs/squashfs.go

```diff
@@ -36,3 +36,27 @@ func GetPath() (string, error) {
 	// exec.LookPath functions on absolute paths (ignoring $PATH) as well
 	return exec.LookPath(p)
 }
+
+func GetProcs() (uint, error) {
+	configFile := buildcfg.SINGULARITY_CONF_FILE
+	c, err := singularityconf.Parse(configFile)
+	if err != nil {
+		return 0, fmt.Errorf("unable to parse singularity.conf file: %s", err)
+	}
+	// proc is either "" or the string value in the conf file
+	proc := c.MksquashfsProcs
+
```

Comment:
Just for information, ideally it would be great to follow the logic commented here https://github.com/sylabs/singularity/issues/1228#issuecomment-575715043 for cpus. Not a change request just to let you know what changes will come after merging this PR